### PR TITLE
Kurl proxy in kotsadm project

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

File: kurl_proxy/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+
```
Comment (areed) [inline kurl_proxy/README.md]: These instructions won't work

**Comment by moonlight16** [comment]
I'm just curious.. what's the purpose of moving proxy servers into the kotsadm repo?

**Comment by emosbaugh** [comment]
@moonlight16 the original goal is to version the kurl-util image for reproducible installs. the kurl-util image is made up of 3 binaries, this being one of them. i discussed with marc and we believe that the proxy binary is more appropriate as part of the kotsadm project. now what to do with the others...

**Comment by emosbaugh** [comment]
@areed you would need to clone the kurl project as well and follow those instructions

**Pull request merged**